python/bootstrap.py

```diff
@@ -383,6 +383,69 @@ def setup_projects(backend):
                                  headers={'Content-type': 'application/json'},
                                  auth=(FUSION_USERNAME, FUSION_PASSWORD))
   print ("Finished creating datasource")
+
+# Changes on following files
+# __init__.py
+# line 39 added parser arg
+#
+# ./autocorrect_config/grouped_query_autocorrect_pipeline.json
+# line 2,12,16,76,84
+#
+# ./autocorrect_config/lucidfind-autocorrect-indexer_datasource.json
+# line 2,7,35,40,41
+
+# begining of autocorrect module
+if cmd_args.activate_autocorrect_typeahead:
+  # collection id changed and the folder from where the files are loaded. Rest is the same.
+  collection_id = "website-typeahead"
```

Comment:
Changed the collection name and pipeline name too. So if we have to reset Fusion, deleting the collection will automatically remove the pipeline too.